homeassistant/components/plugwise/climate.py

```diff
@@ -145,14 +144,6 @@ def preset_mode(self) -> str | None:
         """Return the current preset mode."""
         return self.device.get("active_preset")
 
-    @property
-    def extra_state_attributes(self) -> Mapping[str, Any] | None:
-        """Return entity specific state attributes."""
-        return {
-            "available_schemas": self.device["available_schedules"],
-            "selected_schema": self.device["select_schedule"],
```

Comment:
As a `Number` entity I can only see `maximum_boiler_temperature`.
So they are not numbers?